### PR TITLE
feat(inscription): add inscription to the address card

### PR DESCRIPTION
**Pull request by Daryl-L**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/516

### CODE REVIEW

File: src/pages/Address/AddressComp.tsx [hunk 1]
```diff
@@ -261,32 +193,70 @@ export const AddressOverviewCard: FC<{ address: Address }> = ({ address }) => {
 
       <CardCellsLayout type="leftSingle-right" cells={overviewItems} borderTop />
 
-      {udtAccounts.length || cotaList?.length ? (
+      {udts.length > 0 || (cotaList?.length && cotaList.length > 0) || inscriptions.length > 0 ? (
```
Comment (Keith-CY) [inline src/pages/Address/AddressComp.tsx:196]: What's the difference between `udts.length` and `udts.length > 0`

**Comment by Keith-CY** [comment]
This address received an unknown sudt, but it's not listed
https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2
image

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
image

Underline of tab is not aligned

page: https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qqfy5cxd0x0pl09xvsvkmert8alsajm38qfnmjh2fzfu2804kq47vqdhk85uk74t6shc4ap7dlh94qrhy795jngsqrm48

File: src/pages/Address/AddressComp.tsx (hunk 1 above)
Comment (Daryl-L) [reply src/pages/Address/AddressComp.tsx:196]: > What's the difference between `udts.length` and `udts.length > 0`

No difference, I think maybe it is more semantical to read the code "the length is larger than 0"

**Comment by Daryl-L** [comment]
> This address received an unknown sudt, but it's not listed https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2 image
> 
> It's not listed because the token info is missing



> This address received an unknown sudt, but it's not listed https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2 image
> 
> It's not listed because the token info is missing



> image
> Underline of tab is not aligned
> 
> page: https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qqfy5cxd0x0pl09xvsvkmert8alsajm38qfnmjh2fzfu2804kq47vqdhk85uk74t6shc4ap7dlh94qrhy795jngsqrm48

I have found this problem, just in progress to fix it.

**Comment by Daryl-L** [comment]
> > This address received an unknown sudt, but it's not listed https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2 image
> > It's not listed because the token info is missing
> 
> > This address received an unknown sudt, but it's not listed https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2 image
> > It's not listed because the token info is missing
> 
> > image
> > Underline of tab is not aligned
> > page: https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qqfy5cxd0x0pl09xvsvkmert8alsajm38qfnmjh2fzfu2804kq47vqdhk85uk74t6shc4ap7dlh94qrhy795jngsqrm48
> 
> I have found this problem, just in progress to fix it.

Wrong comment to reply, the problem I said was this https://github.com/Magickbase/ckb-explorer-frontend/pull/208#issuecomment-1892997436.

And here, all the items from list was listed here, maybe @zmcNotafraid could have a lock.

**Comment by Keith-CY** [comment]
> > > This address received an unknown sudt, but it's not listed [ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2](https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2) image
> > > It's not listed because the token info is missing
> > 
> > 
> > > This address received an unknown sudt, but it's not listed [ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2](https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2) image
> > > It's not listed because the token info is missing
> > 
> > 
> > > image
> > > Underline of tab is not aligned
> > > page: https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qqfy5cxd0x0pl09xvsvkmert8alsajm38qfnmjh2fzfu2804kq47vqdhk85uk74t6shc4ap7dlh94qrhy795jngsqrm48
> > 
> > 
> > I have found this problem, just in progress to fix it.
> 
> Wrong comment to reply, the problem I said was this [#208 (comment)](https://github.com/Magickbase/ckb-explorer-frontend/pull/208#issuecomment-1892997436).
> 
> And here, all the items from list was listed here, maybe @zmcNotafraid could have a lock.

I've compared it with the page on testnet and found the `unknown` sudt tokens are all not listed.
Then I visited an address which holds some known sudt tokens and they are listed, so I think it's by design

**Comment by Daryl-L** [comment]
> This address received an unknown sudt, but it's not listed https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qrfrwcdnvssswdwpn3s9v8fp87emat306ctjwsm3nmlkjg8qyza2cqgqqypkqx2llypfkk95j65eguhflr9htpxtegp9rdm2 image
> 
> It's not listed because the token info is missing

done

**Comment by Daryl-L** [comment]
> image
> Underline of tab is not aligned
> 
> page: https://ckb-explorer-frontend-in-magickbase-repo-git-tab-magickbase.vercel.app/address/ckt1qqfy5cxd0x0pl09xvsvkmert8alsajm38qfnmjh2fzfu2804kq47vqdhk85uk74t6shc4ap7dlh94qrhy795jngsqrm48

done

**Comment by Keith-CY** [comment]
@FrederLu please have a review

**Comment by FrederLu** [comment]
https://ckb-explorer-frontend-in-magickbase-repo-iqd7orzo6-magickbase.vercel.app/address/ckt1qrejnmlar3r452tcg57gvq8patctcgy8acync0hxfnyka35ywafvkqgp0s624ekepnj9kcezh4vurxg5z0w0zakaqg3cnp0f?tx_status=committed

1、`User Defined Tokens` copy needs to be updated.
image
image

2、The style appears abnormal when the inscription is long.
image


3、When the page is displayed in Chinese, some styles need to be adjusted.
image

**Pull request merged**